notebooks/samples/explanations/ai-explanations-image-tf2.ipynb

```diff
@@ -0,0 +1,1758 @@
+{
```

Comment:
<p>In order to deploy this model to AI Explanations, you need to create an <code>explanation_metadata.json</code> file with information about our model inputs, outputs, and baseline. </p><p>For image models, using <code>[0,1]</code> as your input baseline represents black and white images. This example uses <code>np.random</code> to generate the baseline because the training images contain a lot of black and white (i.e. daisy petals). </p><p> </p><p>Note: for the explanation request, use the model's signature for the input and output tensors. Do not use the serving function signature.</p><p> </p><p> </p><p>(Side question: why did we query the signature of the serving input function layer in the previous section, when we're explicitly not using it in the explanation metadata file?)</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/192/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>